index.html

```diff
@@ -1,27 +1,51 @@
-        </div>
-
-        <div class="button">
-            <button>Sign In</button>
-        </div>
+   
+   <!--------- hero title ---------->
+   <section class="section-center">
+      <article>
+         <h1 class="hero-text">Unlimited movies, TV shows, and more</h1>
+         <p class="sub-text">Watch anywhere. Cancel anytime.</p>
+      </article>
+   </section>
 
-        </nav>
-    </header>
 
- 
+   <!--------- email address ---------->
+   <section class="section-center">
+      <div class="email-address">
+         <article class="email-title">
+            <h2>Ready to watch? Enter your email to create or restart your membership.</h2>
+         </article>
+         <div class="email-form">
+            <form action="#">
+               <input type="email" name="email" class="form-control" placeholder="Email address" required>
+            </form>
+            <div>
+               <button class="btn" type="submit">
+               get started<i class="uil uil-angle-right-b"></i></i>
```

Comment:
For this project let's start with regular use of svg icons before moving to libraries, find a right icon svg, download it, create a folder in the project called icons and store it there and then use it here